src/utils/keyprovider.rs

```diff
@@ -12,6 +12,7 @@ pub struct KeyProviderKeyWrapProtocolOutput {
 pub mod key_provider_service_client {
     #![allow(unused_variables, dead_code, missing_docs, clippy::let_unit_value)]
     use tonic::codegen::*;
+    use tonic::codegen::http::Uri;
```

Comment:
Not strictly part of this PR, but it would be super helpful if the *top* of this file mentioned it was auto-generated (as we do in Kata [here](https://github.com/kata-containers/kata-containers/blob/main/src/libs/protocols/build.rs#L122)).

Reply:
Because the release deadline is approaching and i am not familiar with `tonic-build`, I prefer to propose a new PR to refine the auto-generated file. Maybe I can make the code about proto more elegant. :D